src/components/core/DpBulkEditHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     role="menu"
-    class="layout__item u-12-of-12 u-p-0_5 bg-color--system-info-light-1 line-height--1_6">
+    class="layout__item u-12-of-12 u-p-0_5 bg-color-ui-selection line-height--1_6">
```

Comment:
As You point me to think about it @spiess-demos I actually would omit the `ui-` but this is just my personal feeling.

Reply:
I updated utility classes in https://github.com/demos-europe/demosplan-core/pull/1326 and here, in b2683da3a95a7c6897591616f466078647654bd1